### ISSUE TITLE
error localhost:8080 when running 'npm run watch'

### ISSUE THREAD

**Issue by dokenzy** [ufocoder/redux-universal-boilerplate#8]
`npm run build` and `npm start` works well.

`npm run watch` runs without any errors or warnings itself, but open `localhost:8080` in web browser, I get a error message:

`Error: ENOENT: no such file or directory, stat '/Users/account/git/another-react-starter/dist/index.html'
   at Error (native)`
It is really weird because the path is not existed anymore: `/Users/account/git/another-react-starter/`.

One more thing(next week, the Apple's event :p)

Opening `localhost:8080`, there is a 404 error: assets/client.js

I'm using Mac(El Capitan), node=6.4.0, npm=3.10.3.

Thanks.

**Comment by ufocoder** [comment]
@dokenzy Thanks for issue, this was fixed by 1dba699012e4cfebab5169bc1d3fe43bc38b42ad

**Comment by ufocoder** [comment]
@dokenzy Boilerplate build server and client bundlers, in your situation server bundle in `watch` mode was build successfully, but not for client bundle. 

You create this issue after introduce bad webpack config for `prod` mode, what brakes client bundle building (perhaps this problem behavior depend on #7), but it's fixed now

Please check your bundle building now and give me a feedback. Thanks

**Comment by dokenzy** [comment]
Thank you for respnse.

Pulling your latest commit, prod mode runs well but this issue is not resolved and `git/redux-universal-boilerplate/static/assets` is empty still.

I can't understand the error message:

Error: ENOENT: no such file or directory, stat '/Users/account/git/**_another-react-starter_**/dist/index.html'   at Error (native).

`another-react-starter` directory was removed a few days ago. 

The reason could be my environment not your code, but I can't guess the cause.

Thanks.

**Comment by ufocoder** [comment]
I've got your issue, watch mode store client bundle assets in memory, it's a right way for fast development, if you have to use assets in your component check home page component https://github.com/ufocoder/redux-universal-boilerplate/tree/master/src/common/components/Page/Home

Don't require builded assets directly

**Comment by dokenzy** [comment]
strange. I don't edit the code.

 BTW, what makes the path "another-react-start"? Do you think this is related to `createMemoryHistory` in react-router? Please let me know how to fix this.

Thanks.

**Comment by ufocoder** [comment]
I think it was maked by git when you cloned a repo :-D

**Comment by dokenzy** [comment]
No, I cloned this repo as 'redux-universal-boilerplate',  I tested another react boilerplat as 'another-react-starter' from another repo, and deleted the directory a few days ago. 'another-react-starter' directory doesn't exist.

After your last commit, I deleted 'redux-universal-boilerplate' directory totally, and cloned again as the same name 'redux-universal-boilerplate' in another directory, but the problem and error message is still the same.
